lib/ash_ai/mcp/dev.ex

```diff
@@ -1,18 +1,39 @@
 defmodule AshAi.Mcp.Dev do
-  @moduledoc "Place in your endpoint's code_reloading section to expose Ash dev MCP at /mcp/ash"
+  @moduledoc """
+  Place in your endpoint's code_reloading section to expose Ash dev MCP"
+
+  Default path is `/mcp/ash`
+  """
   @behaviour Plug
 
   @impl true
   def init(opts) do
-    AshAi.Mcp.Router.init(Keyword.put(opts, :tools, :ash_dev_tools))
+    path =
+      opts
+      |> Keyword.get(:path, "/mcp/ash")
+      |> String.split("/")
+      |> Enum.reject(&(&1 == ""))
+
+    opts =
+      opts
+      |> Keyword.put(:tools, :ash_dev_tools)
+      |> Keyword.put(:path, path)
+
+    AshAi.Mcp.Router.init(opts)
   end
 
   @impl true
-  def call(%Plug.Conn{path_info: ["mcp", "ash" | rest]} = conn, opts) do
-    conn
-    |> Plug.forward(rest, AshAi.Mcp.Router, opts)
-    |> Plug.Conn.halt()
-  end
+  def call(%Plug.Conn{path_info: path_info} = conn, opts) do
+    expected_path = Keyword.get(opts, :path)
+
+    if List.starts_with?(path_info, expected_path) do
+      rest = Enum.drop(path_info, length(expected_path))
```

Comment:
Not sure if there is a more performant way to do this, because the length of the path is not fixed

Reply:
We could do:
```elixir
case Enum.split(path_info, length(expected_path)) do
  {^expected_path, rest} ->
       conn
       |> Plug.forward(rest, AshAi.Mcp.Router, opts)
       |> Plug.Conn.halt()
    _ ->
       conn
end
```